dask/dataframe/tests/test_dataframe.py

```diff
@@ -3179,6 +3189,7 @@ def test_corr():
     pytest.raises(TypeError, lambda: da.corr(ddf))
 
 
+@pytest.mark.xfail(reason="Dataframe does no longer support duplicate column names")
```

Comment:
I would use xfail for cases that *should* pass, but we know they currently don't, i.e., a situation that might get fixed. Here we are purposefully removing a capability, never to return, so the test should be removed.